content/en/contribute/code/style-guide.md

```diff
@@ -6,25 +6,38 @@ description: >
   Guidelines for writing code
 ---
 
+## Language
+
+Our strong preference is to use JavaScript or TypeScript wherever possible, including in webapps, on the server, and for scripting. This is because:
```

Comment:
Small wording change. My thinking here is that rather than state "this is how we do it", which someone could read and think, "ok, whatever, that's nice, you have a nice time with that" the suggested wording is more instructive on what we are asking you to do.

```suggestion
Prefer JavaScript or TypeScript wherever possible, including in webapps, on the server, and for scripting. This is because:
```